salt/vault/files/vault.conf.jinja

```diff
@@ -0,0 +1,24 @@
+{%- from "vault/map.jinja" import vault with context -%}
```

Comment:
While these imports _work_ like this in templated files, it may be cleaner/less expensive to import _once_ in the sls file (which you already do), then pass the `context` vars through `file.managed`...

Reply:
Makes sense...updated to use jinja templating.